Add migration notes for subtitles as tracks (#4844)
opencast/opencast: docs/guides/admin/docs/releasenotes/subtitles-as-first-class-citizens.txt

> - With OC14 we change a lot how subtitles work in Opencast, you may need to migrate!

Reviewer: ```suggestion
- With OC 15 we change a lot how subtitles work in Opencast, you may need to migrate!
```

Subtitles as first class citizens
---------------------------------------

> With Opencast 14 we want to put more emphasis on subtitles. You can find more details on how subtitles should be

Reviewer: ```suggestion
With Opencast 15 we want to put more emphasis on subtitles. You can find more details on how subtitles should be
```

handled going forward in [Subtitles](../modules/subtitles.md).

> This comes with a bit of migration. Namely, subtitles should not be stored as "attachments" anymore, but as "media"
> (as they is called in the Admin UI) or "tracks" (as the are called internally). Therefore, all subtitle files currently
> stored as attachments in your events should be moved to tracks. This can easily be accomplished with the "changetype"

Reviewer 2: ```suggestion
This comes with a bit of migration. Namely, subtitles should not be stored as "attachments" or "catalogs" anymore, but as "media"
(as they is called in the Admin UI) or "tracks" (as the are called internally). Therefore, all subtitle files currently
stored as attachments or catalogs in your events should be moved to tracks. This can easily be accomplished with the "changetype"
```

> workflow operation handler new to Opencast 14. See example below. (Subtitles should then be republished)

Reviewer: `Opencast 15.x` instead.

> workflow operation handler new to Opencast 14. See example below. (Subtitles should then be republished)

Reviewer: ```suggestion
workflow operation handler new to Opencast 15. See example below. (Subtitles should then be republished)
```
